contracts/VaultFactory.vy

```diff
@@ -172,15 +170,44 @@ def protocol_fee_config(vault: address = msg.sender) -> PFConfig:
     @return The protocol fee config for the msg sender.
```

Comment:
return values changed, so we may want to update to:
```
@return Fee in bps
@return Address of fee recipient
```

Reply:
https://github.com/yearn/yearn-vaults-v3/pull/211/commits/abf3e20af1b37397efaa8569934ee92889cea712